docs/data-sources/cluster.md

```diff
@@ -28,6 +28,7 @@ data "cockroach_cluster" "cockroach" {
 ### Read-Only
 
 - `account_id` (String) The cloud provider account ID that hosts the cluster. Needed for CMEK and other advanced features.
+- `backup_config` (Attributes) (see [below for nested schema](#nestedatt--backup_config))
```

Comment:
This is all generated automatically.  We could customize it, but I'm unclear if we could do it in a way that wouldn't provide ongoing maintenance.  It may be possible but It would need to be considered for a separate project and would need to be prioritized.

Reply:
Got it, makes sense!